app/javascript/packages/document-capture/hooks/use-if-still-mounted.js

```diff
@@ -19,11 +19,16 @@ function useIfStillMounted() {
     };
   });
 
-  const ifStillMounted = (fn) => (...args) => {
-    if (isMounted.current) {
-      fn(...args);
-    }
-  };
+  /**
```

Comment:
This change isn't strictly relevant or necessary for this work, but it was useful for the revisions to `FormSteps` to be able to have proper type definitions on the arguments in the step's `onError` callback.

This also explains the change to `FileImage`, since it became more strict and properly detected an issue in how the state was being initialized in that component.